src/time_integration/paired_explicit_runge_kutta/polynomial_optimizer.jl

```diff
@@ -0,0 +1,117 @@
+using LinearAlgebra: eigvals
+using ECOS
+using Convex
+const MOI = Convex.MOI
+
+function filter_eigvals(eig_vals, threshold)
+    filtered_eigvals_counter = 0
+    filtered_eig_vals = Complex{Float64}[]
+
+    for eig_val in eig_vals
+        if abs(eig_val) < threshold
+            filtered_eigvals_counter += 1
+        else
+            push!(filtered_eig_vals, eig_val)
+        end
+    end
+
+    println("$filtered_eigvals_counter eigenvalue(s) are not passed on because they are in magnitude smaller than $threshold \n")
+
+    return length(filtered_eig_vals), filtered_eig_vals
+end
+
+function polynoms(cons_order, num_stage_evals, num_eig_vals,
```

Comment:
Maybe we should rename this to `stability_polys` or `stability_polynomials` to avoid possible ambiguity with another `polynomials` function (that might be already present or could be added later)